AppCreationScripts/sample.json

```diff
@@ -15,7 +15,7 @@
     {
       "Id": "client",
       "Name": "iwa-console",
-      "Kind": "Desktop",
+      "Kind": "Console",
```

Comment:
@kalyankrishna1 : "Console" is not present in the ApplicationKind in the generator